code/modules/mob/living/carbon/xenomorph/castes/puppet/puppet.dm

```diff
@@ -0,0 +1,218 @@
+#define WITHER_RANGE 15
+
+// THIS SHOULD NEVER BE IN PLAYER CONTROL (like actually having a client)
```

Comment:
Why?

Reply:
it tends to shit itself whenever a player controls it

Reply:
What does that mean

Reply:
tends to not work even

Reply:
How does it tend not to work? this should work fine from what I see